engine/src/flutter/tools/pkg/engine_build_configs/lib/src/build_config_runner.dart

```diff
@@ -518,10 +523,7 @@ final class BuildRunner extends Runner {
         pid: result.pid, // pid,
       );
     }
-    String okMessage = bootstrapResult.stdout.trim();
```

Comment:
There are some tests of this code in https://github.com/flutter/flutter/blob/master/engine/src/flutter/tools/pkg/engine_build_configs/test/build_config_runner_test.dart.

Is there any way to write a test that would fail if the calls were out of order?

Reply:
Added a test